bot/cogs/tags.py

```diff
@@ -86,11 +86,60 @@ def _get_suggestions(self, tag_name: str, thresholds: Optional[List[int]] = None
             return self._get_suggestions(tag_name)
         return found
 
+    async def _get_tags_via_content(self, check: callable, keywords: str) -> Optional[Embed]:
```

Comment:
`Callable[[Iterable], bool]` would be a more accurate annotation.

Reply:
Nice, thank you, I've never used `Callable` from `typing` before!